routes/user.js

```diff
@@ -26,11 +26,11 @@ router.post('/register', (req, res) => {
 
 // login user --------------------------------------------------------------- //
 
-router.get('/users/login', (req, res) => {
-  res.render('users/login');
+router.get('/login', (req, res) => {
+  res.render('index');
```

Comment:
this should render a template called login.pug not an index.pug your template names matter.